dali/base/dasess.cpp

```diff
@@ -938,8 +938,14 @@ class CClientSessionManager: public CSessionManagerBase, implements IConnectionM
         }
 
 
+
         if (!queryCoven().sendRecv(mb,RANK_RANDOM,MPTAG_DALI_SESSION_REQUEST,SESSIONREPLYTIMEOUT))
+        {
+            StringBuffer user;
+            udesc->getUserName(user);
+            PROGLOG("Dali call to get user '%s' '%s' permissions failed. Returning SecAccess_None for '%s')", user.str(), nullText(key), nullText(obj));
```

Comment:
This can only happen if there's a (>3 minute) timeout communicating to Dali..

Surely the message should clarify the cause?
As it stands this text sounds like a normal denial.